### PR TITLE
WIP: Re-enabling S3 storage for Fedora 6

### PR DESCRIPTION
**Pull request by noahwsmith**
Per #129 it appears that only F5 can write to S3, so I have started the process of enabling F6 to write to S3.

### CODE REVIEW

File: fcrepo6/rootfs/opt/tomcat/bin/setenv.sh [hunk 1]
```diff
@@ -32,3 +32,10 @@ fi
 if [[ "${FCREPO_DISABLE_SYN}" == "true" ]]; then
   export CATALINA_OPTS="${CATALINA_OPTS} -Dfcrepo.properties.management=relaxed"
 fi
+
+if [[ "${FCREPO_BINARYSTORAGE_TYPE}" == "file" ]]; then
+  export CATALINA_OPTS="${CATALINA_OPTS} -Dfcrepo.storage=ocfl-fs"
+fi
+if [[ "${FCREPO_BINARYSTORAGE_TYPE}" == "s3" ]]; then
+  export CATALINA_OPTS="${CATALINA_OPTS} -Dfcrepo.storage=ocfl-s3 -Dfcrepo.aws.region=${FCREPO_AWS_REGION} -Dfcrepo.aws.region=${FCREPO_S3_BUCKET} -Dfcrepo.aws.region=${FCREPO_S3_PREFIX}"
```
Comment (nigelgbanks) [inline fcrepo6/rootfs/opt/tomcat/bin/setenv.sh:40]: Does the user name / password not need to be passed in as well?

**Comment by nigelgbanks** [comment]
I don't have the ability to test this at the moment but it looks safe for merging, I'll wait for a yea or neigh :horse: til Monday

**Comment by noahwsmith** [comment]
@emudojo is testing this over the next few days - we'll give you a 👍 when it's ready

File: fcrepo6/rootfs/opt/tomcat/bin/setenv.sh (hunk 1 above)
Comment (noahwsmith) [reply fcrepo6/rootfs/opt/tomcat/bin/setenv.sh:40]: Probably so. Hertzel was testing this today...

**Comment by nigelgbanks** [comment]
> @emudojo is testing this over the next few days - we'll give you a 👍 when it's ready

Oh ok I'll move the pull request into draft in the mean while 👍

**Comment by nigelgbanks** [comment]
Closing as https://github.com/Islandora-Devops/isle-buildkit/pull/134 was opened

**Pull request closed without merging**